apps/ui-library/content/docs/nuxtjs/client.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Supabase Client Libraries
+description: Supabase client for Nuxt.js
+---
+
+## Installation
+
+<BlockItem name="supabase-client-nuxtjs" description="Supabase Client for Nuxt.js" />
+
+## Folder structure
+
+<RegistryBlock itemName="supabase-client-nuxtjs" />
+
+## Usage
+
+This block installs a Supabase client for connecting your Nuxt.js project to Supabase. It's designed to fully supports server-side rendering (SSR).
+
+If you've already set up your Supabase client—either using the `npm create nuxt@latest` template or another method—you can continue using your existing setup.
+
+### Getting started
+
+After installing the block, you'll have the following environment variables in your `.env.local` file:
+
+```env
+NUXT_PUBLIC_SUPABASE_URL=
+NUXT_PUBLIC_SUPABASE_PUBLISHABLE_OR_ANON_KEY=
+```
+
+- If you're using supabase.com, you can find these values in the [Connect modal](https://supabase.com/dashboard/project/_?showConnect=true) under App Frameworks or in your project's [API settings](https://supabase.com/dashboard/project/_/settings/api).
+
+- If you're using a local instance of Supabase, you can find these values by running `supabase start` or `supabase status` (if you already have it running).
+
+- If you wish to have a great Developer Experience while using Supabase with Nuxt, try the official Nuxt [module](https://supabase.nuxtjs.org/).
+
+<Callout type="warning" className="mt-4">
+  {' '}
+  This Supabase client is built for SSR with the Nuxt.js. If you're building a Vue SPA, use the [Vue
+  SPA client](/ui/docs/vue/client) instead.{' '}
+</Callout>
+
+## Further reading
+
+- [Generating TypeScript types for your client](https://supabase.com/docs/guides/api/rest/generating-types)
```

Comment:
Are these resources good? Let's only include them if they're current and helpful.